context/package-manager.json

```diff
@@ -21,11 +21,6 @@
       "svg",
       "ttf",
       "json"
-    ],
```

Comment:
Can we leave this in place as this is just saying that a test folder is allowed, which is still true